test/e2e/e2ev1.go

```diff
@@ -30,77 +29,62 @@ import (
 
 var _ = Describe("kubebuilder", func() {
 	Context("with v1 scaffolding", func() {
-		imageName := "controller:v0.0.1"
 		var testSuffix string
-		var c *config
-		var kbTest *kubebuilderTest
+		var kbc *KbContext
 
 		BeforeEach(func() {
 			var err error
 			testSuffix, err = randomSuffix()
 			Expect(err).NotTo(HaveOccurred())
-			c, err = configWithSuffix(testSuffix)
+			env := []string{"GO111MODULE=off"}
+			kbc, err = ContextWithSuffix(testSuffix, env)
 			Expect(err).NotTo(HaveOccurred())
-			kbTest = &kubebuilderTest{
-				Dir: c.workDir,
-				Env: []string{"GO111MODULE=off"},
-			}
-			prepare(c.workDir)
+			Expect(kbc.Preparewd()).To(Succeed())
```

Comment:
May be `kbc` should expose `Prepare()` method which just initializes the workdir for now.